packages/cosmwasm/src/multisigcosmwasmclient.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { BroadcastMode, GasLimits, GasPrice, OfflineSigner } from "@cosmjs/launchpad";
+
+import { CosmWasmFeeTable, ExecuteResult, SigningCosmWasmClient } from "./signingcosmwasmclient";
+
+export type Duration =
```

Comment:
Why is this called "duration"? Isn't this a point in time, not a time span?

Reply:
Sorry, this should be `Expiration`.